models/calendar.yml

```diff
@@ -319,10 +319,10 @@
         title: Introductions
         emoji: 👋
 
-    - 7:45 PM:
+    - 7:45 PM:  
         type: Talk
         emoji: 🎨
-        title: TBD
+        title: Tech Feminism 101: Why We Should All Be Feminists
```

Comment:
Whee! Okay should be fixed now.


On Thu, Aug 11, 2016 at 4:21 PM, Billy Roh <notifications@github.com> wrote:

> In models/calendar.yml
> <https://github.com/wafflejs/wafflejs.github.io/pull/189#discussion_r74521502>
> :
>
> >          type: Talk
> >          emoji: 🎨
> > -        title: TBD
> > +        title: Tech Feminism 101: Why We Should All Be Feminists
>
> Also, could you add a description for your talk? Thank you!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wafflejs/wafflejs.github.io/pull/189/files/155ce6247aa36f8a16ace49c009753f06f41c954#r74521502>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABuT3cNsJoCZ-Vxas4Ae_z_F2turpjV1ks5qe66UgaJpZM4JimfE>
> .
>



-- 
Ainsley Wagoner | ainsleywagoner.com
freshwash <http://freshwash.tumblr.com/>